UI/data/locale/en-US.ini

```diff
@@ -985,6 +985,9 @@ Basic.Settings.Stream.MultitrackVideoStreamDumpEnable="Enable stream dump to FLV
 Basic.Settings.Stream.MultitrackVideoConfigOverride="Config Override (JSON)"
 Basic.Settings.Stream.MultitrackVideoConfigOverrideEnable="Enable Config Override"
 Basic.Settings.Stream.MultitrackVideoLabel="Multitrack Video"
+Basic.Settings.Stream.SimulcastLabel="Simulcast"
+Basic.Settings.Stream.SimulcastInfo="Simulcast allows you to encode and send multiple video qualities. No information about your computer and software setup will be sent."
```

Comment:
We can probably lose the extra shade here. It doesn't tell the user about what the feature does

Reply:
@DDRBoxman This is sincere. If users only know RTMP I would like them to know it different.

They control the settings and shouldn’t expect the user to make any changes.

Reply:
Easiest thing to do is just remove the extra sentence. Debatable that it could describe that it may increase system resources or something else useful to the end-user enabling it, but that can be adjusted/added later if it comes up.

Reply:
@Fenrirthviti Done! Second sentence is deleted